source/recurring_payments/index.html.md.erb

```diff
@@ -0,0 +1,389 @@
+}
+```
+
+`set_up_agreement` tells our API to save your user’s card details when they complete this payment. The user’s card details are saved to the agreement with the ID you send.
+
+You can also use `set_up_agreement` to reactivate a cancelled agreement.
+
+When your user is making their first payment, GOV.UK Pay will show your user the `description` you sent when you created the agreement.
+
+Once the user has made the payment, the agreement’s `status` will be `active`. You can use active agreements to take recurring payments.
+
+If the user’s payment fails, their payment details are not saved and the agreement’s `status` remains `created`.
+
+Read more about creating payments in [our guidance for taking payments](/making_payments) or our [API reference page for creating payments](/api_reference/create_a_payment_reference).
+
+## Taking recurring payments
+
+Once you’ve set up the agreement, you can use it to take recurring payments without the paying user interacting with your service.
+
+You must tell your user that you are going to take a payment.
+
+To take a recurring payment, create a payment through the API (`POST v1/payments`) and include the following parameters in the request body:
+
+* `"authorisation_mode" : "agreement"`
+* `"agreement_id": "{AGREEMENT_ID}"`
+
+Replace `{AGREEMENT_ID}` with the `agreement_id` you just set up.
+
+Your request should look something like this:
+
+```json
+{
+  "amount": 1000,
+  "description": "Council tax payment for 2022/23",
+  "authorisation_mode": "agreement",
+  "agreement_id": "cgc1ocvh0pt9fqs0ma67r42l58"
+}
+```
+
+The API returns a standard create payment response. If your request succeeds, the response includes the `agreement_id` you sent and `authorisation_mode` is `agreement`.
+
+GOV.UK Pay uses a token representing the user’s saved payment details from the agreement to automatically take the payment.
+
+If a recurring payment fails when we try to take it, it’s handled like any other failed payment. You can use [GOV.UK Pay’s webhooks to receive an automatic update](/recurring_payments/#use-webhooks-to-receive-automatic-event-updates) when a recurring payment fails.
+
+You should let your users know when you plan to take a payment.
+
+Read more about creating payments in [our guidance for taking payments](/making_payments) or our [API reference page for creating payments](/api_reference/create_a_payment_reference).
+
+### Use idempotence to avoid double-charging a user
+
+Idempotence in an API is when multiple identical requests have the same effect as making a single request. In GOV.UK Pay, you can avoid double charging users by making idempotent requests when creating recurring payments.
+
+Idempotence is useful if you create a recurring payment through our API but do not receive a response due to network failure. With idempotence, further payment creation requests fail. Without idempotence, further payment creation requests will succeed and you may double charge your user.
```

Comment:
Might be useful to emphasise that they do not receive a response so it’s not knowable whether the request succeeded or not.